op-devstack/presets/twol2.go

```diff
@@ -0,0 +1,56 @@
+package presets
+
+import (
+	"github.com/ethereum/go-ethereum/log"
+
+	"github.com/ethereum-optimism/optimism/op-devstack/devtest"
+	"github.com/ethereum-optimism/optimism/op-devstack/dsl"
+	"github.com/ethereum-optimism/optimism/op-devstack/shim"
+	"github.com/ethereum-optimism/optimism/op-devstack/stack"
+	"github.com/ethereum-optimism/optimism/op-devstack/stack/match"
+	"github.com/ethereum-optimism/optimism/op-devstack/sysgo"
+)
+
+// TwoL2 represents a two-L2 setup without interop considerations.
```

Comment:
Given that this is explicitly "without interop", do we want to assert this behaviour here?
My thinking is that in the future these chains may be Interop-enabled by default.